src/services/policiesService.js

```diff
@@ -92,30 +148,32 @@ const getPolicyResults = async (body) => {
     return responsePresidentes.find((candidato) => item.org_politica_id === candidato.org_politica_id && candidato.cargo_id === cargoId);
   }
 
-  let listaIdPartidosObtenidos = responsePreguntaPartido.map((item) => item.org_politica_id);
-
-  let queryPartidosSinCompatibilidad = "SELECT a.id AS org_politica_id, a.orden_cedula, a.nombre, a.alias, 0 AS total FROM partidos_alias a WHERE a.id NOT IN (?) AND a.id IN (SELECT org_politica_id FROM partido_x_respuesta) ORDER BY a.alias ASC";
+  let queryPartidosSinCompatibilidad = "SELECT a.id AS org_politica_id, a.orden_cedula, a.nombre, a.alias FROM partidos_alias a WHERE a.id NOT IN (SELECT org_politica_id FROM partido_x_respuesta) ORDER BY a.alias ASC";
```

Comment:
Hola @dmarquinah, en esta línea me parece que está obteniendo todos los partidos que no están en partido_x_respuesta, pero creo que debería estar más bien limitado a los que aparecen allí, tal como estaba originalmente.

Reply:
Tienes razón, se me fue el NOT, ahora lo reviso